app/src/main/java/de/westnordost/streetcomplete/quests/QuestModule.kt

```diff
@@ -117,13 +117,12 @@ object QuestModule
         AddRecyclingType(o),
         AddRecyclingContainerMaterials(o, r),
         AddSport(o),
-        AddRoadSurface(o, r), // used by BRouter, OsmAnd, OSRM, graphhopper, HOT map style
+        RoadSurface(o, r), // used by BRouter, OsmAnd, OSRM, graphhopper, HOT map style
```

Comment:
I know that the new road surface quest is not always limited to adding the surface but in case of paved etc also modifies it, but I'd rather leave the name as `AddRoadSurface` because
1. `Add`-something has somewhat established itself as the naming standard for quest types. `RoadSurface` sounds like it is a data class or enum, much like `Surface`
2. Since resurvey quests, many quests do not only add information but also modify it, but I didn't change the naming because of that

Reply:
name changed